templates/make_manifest

```diff
@@ -7,6 +7,7 @@ STEMCELL_OS=${STEMCELL_OS:-ubuntu}
 
 infrastructure=$1
 servers=$2
+cf_manifest=$3
```

Comment:
What values are being pulled from the CF manifest for this? Is it something we can set placeholders for and have people override in their stubs?